.github/workflows/lint.yml

```diff
@@ -9,7 +9,4 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
       - uses: psf/black@stable
-      - uses: DoozyX/clang-format-lint-action@v0.12
-        with:
-          source: './src'
-          clangFormatVersion: 12
+
```

Comment:
Double newline

Reply:
Argh, I thought about wait linus wants a trailing new line here

Reply:
It isn't a double new line there just wasn't a trailing new line before this :red_circle:  :penguin:

Reply:
:grin: